docker/nginx/README.md

```diff
@@ -0,0 +1,23 @@
+## Docker image that compiles NGINX from source
+
+The nginx Headers More module doesn't currently accept wildcard matches
+for clearing input headers (it's on a branch somehwere). So if we want to use this
+capability we need to compile nginx from source and add this module.
+
+We're building a custom docker image from a branch of the [Headers More](https://github.com/openresty/headers-more-nginx-module) module. This dockerfile installs the tools necessary
+to compile nginx and the headers more module from source, then copies in our
+custom nginx config.
```

Comment:
I think you can remove the ", then copies in our custom nginx config." part here -- that's still being mounted from the kubernetes configmap.